client/lock.go

```diff
@@ -251,9 +251,9 @@ func (c *lockClient) waitDelete(pfx string, myRev int64) {
 				}
 			}
 			if f {
```

Comment:
It is better a give `f` a full name.

Reply:
It seems you want break the outer for loop, try and a mark `outer` at L245 and use `break outer` ?

Reply:
You're right. I've made the changes you suggested.